tests/e2e/lib.sh

```diff
@@ -325,6 +328,13 @@ deploy_sensor() {
 deploy_sensor_via_operator() {
     local sensor_namespace=${1:-stackrox}
     local central_namespace=${2:-stackrox}
+    local central_namespace=${1:-stackrox}
```

Comment:
These two lines look wrong together.

Reply:
Hah, indeed, thank you. Fortunately the operator-based tests with different namespaces are in a different PR. Will check.